Gruntfile.js

```diff
@@ -48,22 +48,7 @@ module.exports = function (grunt) {
 						util.format('cp %s/LICENSE %s', __dirname, version_dir),
 						util.format('mkdir -p %s/sql/migrations', version_dir),
 						util.format('cp %s/sql/*.sql %s/sql/', __dirname, version_dir),
-						util.format('cp %s/sql/migrations/*.sql %s/sql/migrations/', __dirname, version_dir),
```

Comment:
Migrations?

Reply:
Yes, migrations are copied to release folder.